scripts/mixins/dynamis_beastmen.lua

```diff
@@ -6,7 +6,7 @@ require("scripts/globals/status")
 
 g_mixins = g_mixins or {}
 
-g_mixins.dynamis_beastmen = function(mob)
+g_mixins.dynamis_beastmen = function(dynamisBeastmenMob)
```

Comment:
how about just `dynaBeastman` to shorten that?

Reply:
All others are somethingMob. We should keep the "Mob" there IMO